CMakeLists.txt

```diff
@@ -5,6 +5,17 @@ else()
     project("mbed TLS" C)
 endif()
 
+include(CMakePackageConfigHelpers)
+
+set(CMAKE_EXPORT_DIR "lib/cmake/mbedtls")
```

Comment:
I added a comment for the CMAKE_EXPORT_DIR variable

Reply:
Thanks for addressing my comments. Regarding this issue, I meant updating this file: https://github.com/ARMmbed/mbedtls/blob/development/README.md in the CMake section. You will notice that the `USE_SHARED_MBEDTLS_LIBRARY` is documented there. The variable that you introduced deserves a paragraph there too. It would be useful to describe, what the variable does and what is its default value.